### ISSUE TITLE
Issue using app with South

### ISSUE THREAD

**Issue by efalconer** [flashingpumpkin/django-socialregistration#62]
Ran into a small issue when trying out South on a site that uses django-socialregistration. When I run syncdb I get the following error:

WindowsError: [Error 3] The system cannot find the path specified: 'C:\Python27\lib\site-packages\django_socialregistration-0.4.3-py2.7.egg\socialregistration\migrations/_._'

After doing a little digging I came across this post which seems to describe the issue:

http://groups.google.com/group/south-users/browse_thread/thread/639e832ad4e71d3d?fwc=1&pli=1

Basically because django-socialregistration has migrations it should be marked as not zip safe in the setup.py file.

**Comment by TylerBrock** [comment]
I'm having the same problem, definitely because its zipped up in an egg, good call!
